__tests__/app.test.js

```diff
@@ -321,3 +321,32 @@ describe("/api/articles/:article_id/comments", () => {
     });
   });
 });
+
+describe("/api/comments/:comment_id", () => {
+  describe("DELETE", () => {
+    test("DELETE: 204 - respond with status code of 204 with no content on successfully deleting a comment", () => {
+      const comment_id = 5;
+      return request(app)
+        .delete(`/api/comments/${comment_id}`)
+        .expect(204)
+        .then(() =>
+          // request GET to check we get a 404 after deleting
+          request(app).get(`/api/comments/${comment_id}`).expect(404)
+        );
+    });
```

Comment:
You are not making a GET request here, you are deleting. Your test should be like this:
` request(app).delete(`/api/comments/${comment_id}`).expect(204)`

Reply:
Resolved in https://github.com/dku7/be-nc-news/pull/19